packages/playwright-core/src/client/DEPS.list

```diff
@@ -1,3 +1,4 @@
 [*]
 ../protocol/
 ../utils/isomorphic
+../mcpBundle.ts
```

Comment:
I can't find where this is used.

Reply:
Good find - it isn't and can't be due to platform.